one/one/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+using namespace std;
+
+#define endl '\n';
```

Comment:
They made a NEWLINE macro, I called mine endl because I recalled in the past using an endl, which you have just linked making mine pointless :p